### ISSUE TITLE
@ character not working on German keyboard due to shortcut

### ISSUE THREAD

**Issue by msakowski** [notable/notable#1519]
- **OS Version**: macos 11.5.1
- **Notable Version**: 1.8.4

### Current behavior

I'm using a German keyboard. In order to write an "@" character, I have to press option+L. This command toggles the line numbers.

### Expected behavior

Don't override characters with internal keyboard shortcuts or make them configurable.

**Comment by Martzn** [comment]
That is true! The issue still exists today after two years of reporting! How to enter a mail address? It is not possible. Please fix the error or describe a working work around.

**Comment by fabiospampinato** [comment]
This should be fixed in beta10 https://github.com/notable/notable-insiders/releases/tag/v1.9.0-beta.10
